thunder/tests/test_pythonex.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+import math
+
+import thunder
+
+
+def _run_cache_symbolic_values(fn, ref_fn, *args):
+    jit_fn = thunder.jit(fn, cache="symbolic values")
+    out = jit_fn(*args)
+
+    out_ref = ref_fn(*args)
+    assert out == out_ref
+
+
+def test_fmod():
+    def foo(a, b):
+        return a % b
+
+    _run_cache_symbolic_values(foo, foo, 2.0, 1.3)
```

Comment:
Q: are these float values random or intentionally chosen?

Reply:
Just arbitrary numbers. :stuck_out_tongue_closed_eyes:

Reply:
I was hoping that I can put this in opinfo. But I needed to plumb the cache option "symbolic_values", I'm not sure if that's a knob that we'd want to add, or maybe wait until we switched to symbolic_values as default cache option?!